### PR TITLE
F/1915 reflect custom config updates

### PR DESCRIPTION
**Pull request by sonofflynn89**
Added support for a site's custom configurations.

Affects the following issues:
https://devtopia.esri.com/dc/hub/issues/1915
https://devtopia.esri.com/dc/hub/issues/2064

### CODE REVIEW

File: src/dcat-us/index.test.ts [hunk 1]
```diff
@@ -53,4 +54,86 @@ describe('generating DCAT-US 1.1 feed', () => {
...
+    expect(chk1.contactPoint).toEqual({ '@type': 'vcard:Contact', fn: 'thervey_qa_pre_a_hub' });
+    expect(chk1.accessLevel).toBe('public');
+    expect(chk1.spatial).toBe('-121.118,38.7754,-119.009,39.359');
+    expect(chk1.theme).toEqual(['geospatial']);
+
+    expect(chk1.distribution).toBeInstanceOf(Array);
+    expect(chk1.distribution.length).toBe(6);
+  });
+
+  it('formats catalog correctly', async function () {
```
Comment (sonofflynn89) [inline src/dcat-us/index.test.ts:90]: Well, that's not supposed to be there 😅

Comment (andrewctate) [reply src/dcat-us/index.test.ts:90]: ???

**Review by andrewctate — APPROVED** [review]
Thank you!

**Review by andrewctate — CHANGES_REQUESTED** [review]
Oh, wait—don't forget the changelog!

File: src/dcat-us/index.test.ts (hunk 1 above)
Comment (sonofflynn89) [reply src/dcat-us/index.test.ts:90]: I thought I had only copied one test, but it looks like the `it('formats catalog correctly')` test got duplicated on accident @andrewctate

**Comment by sonofflynn89** [comment]
Thanks for the reminder @andrewctate! I was about to do it before we got into the whole structural discussion with Thomas, but I forgot 😅.

File: example-app/config/default.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
   "port": "8080",
-  "arcgisPortal": "https://www.arcgis.com"
+  "arcgisPortal": "https://www.mapsqa.arcgis.com"
```
Comment (sonofflynn89) [inline example-app/config/default.json:3]: Reminder to myself to remove this

**Review by andrewctate — APPROVED** [review]
Nice work!

**Pull request merged**